src/user-guide/sharing-access-to-workflows.rst

```diff
@@ -1,148 +1,265 @@
+
+Sites can restrict the permissions which users are allowed to delegate to each
+other and can configure default permissions (see :ref:`site_configuration`).
+
+Authorization is configured by these two configurations:
+
+* :py:attr:`c.CylcUIServer.user_authorization
+  <cylc.uiserver.app.CylcUIServer.user_authorization>` (user configuration)
+* :py:attr:`c.CylcUIServer.site_authorization
+  <cylc.uiserver.app.CylcUIServer.site_authorization>` (site configuration)
+
+.. rubric:: Example:
+
+.. code-block:: python
+
+   # ~/.cylc/uiserver/jupyter_config.py
+
    c.CylcUIServer.user_authorization = {
-       "group:groupA": ["read", "play", "stop"],
-       "user2": ["!stop"]
+       # <user/group>: [<permission>, ...],
+
+       # allow "user1" to monitor my workflows
+       "user1": ["READ"],
+
+       # allow "user2" to monitor and trigger tasks in my workflows
+       "user2": ["READ", "Trigger"],
    }
 
-Providing your site configuration permits you to grant this access,
-this configuration, for ``user2`` (who is a member of system group ``groupA``),
-would result in them having ``read`` and ``play`` access. Permission to stop your
-workflows has been removed so this action will be forbidden.
 
-Access Groups
-^^^^^^^^^^^^^
-For convenience, cylc operations available in the UI have been bundled into
-access groups. These should be capitalized to distinguish from cylc operations.
-
-We currently support ``READ``, ``CONTROL`` and ``ALL`` and to remove permissions
-to operations in these groups, use ``!READ``, ``!CONTROL``, ``!ALL``.
-
-
-.. csv-table:: Access Group Mappings
-   :header: "Operation", "READ", "CONTROL", "ALL"
-
-   "Broadcast", , , "X"
-   "Ext-trigger",, "X", "X"
-   "Hold",, "X", "X"
-   "Kill",, "X", "X"
-   "Message",, "X", "X"
-   "Pause",, "X", "X"
-   "Play",, "X", "X"
-   "Poll",, "X", "X"
-   "Read","X", , "X"
-   "Release",, "X", "X"
-   "ReleaseHoldPoint",, "X", "X"
-   "Reload",, "X", "X"
-   "Remove",, "X", "X"
-   "Resume",, "X", "X"
-   "SetGraphWindowExtent",, "X", "X"
-   "SetHoldPoint",, "X", "X"
-   "SetOutputs",, "X", "X"
-   "SetVerbosity",, "X", "X"
-   "Stop",, "X", "X"
-   "Trigger",, "X", "X"
+Users
+^^^^^
 
+There are three methods of identifying a user to grant access to:
+
+``<username>``
+  Configures permissions for a singe user.
+``group:<groupname>``
+  Configures a user group. For more information, see :ref:`group_support`.
+``*``
+  Configures permissions for any authenticated user (see
+  :ref:`Jupyter Hub authenticators reference <authenticators-reference>`
+  for details).
 
 .. note::
 
-   The ``READ`` access group is shorthand for all read-only operations. At present,
-   this is solely the ``read`` operation, which grants access to GraphQL queries and
-   subscriptions, and enables users to see the workflows in the UI. In future
-   the ``READ`` access group may be extended.
+   Using glob patterns (e.g. ``*``) to pattern match user and group names is
+   not currently supported.
```

Comment:
Saying "e.g. `*`" sort of suggests that the last option above is not yet available
```suggestion
   Using glob patterns to match user and group names is
   not currently supported.
```